### DISCUSSION TITLE
what are some free options to use for this plugin that are also quite effective?

### DISCUSSION THREAD

**Discussion by BlitzVI** [brianpetro/obsidian-smart-connections#579] [Q&A] [answered]
i looked through the docs on openrouter and i tried chatgpt 3.5 (free) and asked it what i did on the 24th and it just made things up so i need help to find if its a mistake with the prompt i made or the ai

**Comment by brianpetro** [comment]
Hi @BlitzVI 

The embedding search, used by the Smart Chat, isn't very good at finding date-specific notes, yet. 

If you have a dated note, you can specify the note using the `[[LINK]]` syntax.

And you can use the chat history, open using the button next to the chat name in the Smart Chat, to view which notes were used as context in your chat.

🌴

**Reply by RealRavens** [reply]
I forgot to check, did you add the "search" feature to smart notes yet that you mentioned a while back? If I remember, it was one of those "tools" you wanted to add. Leveraging Obsidian search, where it can be more precise with our notes then send it to be processed and respond back more accurately.

**Reply by brianpetro** [reply]
Hey @RealRavens 

Given the lack of support for utilizing the native Obsidian search feature in community plugins, I decided it would be best to implement it as an action in the Smart Connect software.

This `keyword-search` action is currently available via the ChatGPT integration.

On a related note, today, I got the Smart Connect actions working [in the Smart Chat](https://twitter.com/wfhbrian/status/1788691711134228741). Along with the many other actions provided by Smart Connect, this keyword search action will soon be available in the early-release version.

🌴

**Reply by RealRavens** [reply]
Neat, can't wait to check it out.

**Comment by dougkeiller** [comment, marked as answer]
The new Openrouter end point (thank you @brianpetro !!) gives you access to a *ton* of LLM models.

Including: a new free model "Meta: Llama 3 8B Instruct (free)"
![image](https://github.com/brianpetro/obsidian-smart-connections/assets/128188043/39207ab7-b7d5-4449-a4ee-1b2d136a2a21)
